apps/docs/src/content/reference/core/use-task.mdx

```diff
@@ -0,0 +1,201 @@
+---
+category: '@threlte/core'
+name: useTask
+sourcePath: 'packages/core/src/lib/hooks/useTask.ts'
+order: 5.21
+---
+
+
+<Tip type="info">
+  Tasks are part of Threlte's *Task Scheduling System*.
+  For details on how to use stages and tasks, see the [scheduling
+  tasks](/docs/learn/basics/scheduling-tasks) page.
+</Tip>
+
+The hook `useTask` is used to create a
+[task](/docs/learn/basics/scheduling-tasks#tasks) that is used to run code on
+every frame.
+
+## Creating an Anonymous Task
+
+In its most basic form, `useTask` takes a function as its first argument. This
+function will be executed on every frame, starting on the next frame and
+receives the delta time representing the time since the last frame as its first
+argument. By default, the created task is added to [Threlte's
+`mainStage`](#default-stages) in an arbitrary order (i.e. without dependencies).
+
+```ts
+const { start, stop, started, task } = useTask((delta) => {
+  // This function will be executed on every frame
+})
+```
+
+It returns an object with the following properties:
+- `start`: A function that starts the task. It will be executed on the next
+  frame. Note that by default a task is started automatically.
```

Comment:
I don't see the `autostart` option mentioned in these docs. Maybe it should be added for newcomers?

Reply:
done ✅